packages/default-reporter/src/reporterForClient/reportUpdateCheck.ts

```diff
@@ -40,6 +40,9 @@ function renderUpdateCommand (opts: { env: NodeJS.ProcessEnv, latestVersion: str
   if (opts.env.COREPACK_ROOT) {
     return `corepack prepare pnpm@${opts.latestVersion} --activate`
   }
+  else if (opts.env.VOLTA_HOME) {
```

Comment:
But this doesn't mean that pnpm was installed with volta. It just means that volta is installed on the system. Maybe we should check the location of pnpm, whether it is inside VOLTA_HOME

Reply:
What do you suggest, run `which node` and check if `.volta` is in the path? More importantly, though, do you think this should even be here or should this be left to volta?

Reply:
No, why do we care where is Node.js installed? I mean checking where pnpm is installed. If it is installed in volta dir, then it should be updated with volta.

Here is how to get the location of the pnpm CLI file: https://github.com/pnpm/pnpm/blob/main/packages/plugin-commands-setup/src/setup.ts#L43